Tests/SentryTests/Integrations/Performance/FramesTracking/SentryFramesTrackingIntegrationTests.swift

```diff
@@ -57,6 +62,17 @@ class SentryFramesTrackingIntegrationTests: XCTestCase {
         XCTAssertNil(Dynamic(sut).tracker.asObject)
     }
     
+    func test_HybridSDKEnables_MeasureFrames() {
+        PrivateSentrySDKOnly.framesTrackingMeasurementHybridSDKMode = true
+        
+        let options = fixture.options
+        options.tracesSampleRate = 0.1
```

Comment:
I'd rather remove this, otherwise the tracker would be installed anyway

Reply:
in this case the tracker would not be installed because of the next line.
But anyway, we don't need this here.